### DISCUSSION TITLE
2023-12-15 - 💰 Salary Sacrifice - Electronic Devices - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#7546] [Chewing the Fat]
Hey SSWers,

Lets talk about Salary Sacrifice!
https://ssw.com.au/rules/salary-sacrifice-electronic-devices/

**Comment by camillars** [comment]
great video and explanation from Stephan, and also great to see there’s now an option for people to request a skin too and spread our branding (especially SophieBot)! 💖😍

**Comment by leehawkinsssw** [comment]
A good (and simple) explanation of the benefits of salary sacrificing. 

Stephan's debut on the SSW stage was excellent, well done @stephanfakossw 👏

**Comment by Calinator444** [comment]
I like the idea, but what if you salary sacrifice a device and then leave the company? Wouldn't it be a little strange having the logo of a former employer on your device?

**Reply by leahy268** [reply]
You can just rip them off.  Might need an alcohol wipe to get rid of the last of the residue.  However the bonus is that they protect the top of your laptop from scratches so it will look brand new

**Comment by AlexRothwell** [comment]
Very well explained, including the eligible devices and why an employer might offer salary sacrificing on some devices (they're FBT exempt)

**Reply by jimmy1992abc** [reply]
Hi Alex. because only certain devices are exempt from FBT as per ATO, means there are no extra cost and admin. You can do some research on FBT to understand the defination.

**Comment by jakebayliss** [comment]
This video is 1 month too late... :(

**Comment by jeoffreyfischer** [comment]
I'm not an accountant, but at 1:07, 100k-26k=74k, not 76k 🤔
But again, I'm not an accountant 🫠

**Reply by AntPolkanov** [reply]
Everyone would love such math to be applied to them 🙂

**Reply by stephanfakossw** [reply]
I wished that too 😊💰
Thanks, we're fixing the typo...

**Comment by 2fernandez** [comment]
❤️ Awesome explanation by Stephan with very clear audio

**Would I Salary Sacrifice?**
Not for a $5000 device.

**Reply by wicksipedia** [reply]
why not?

**Reply by jernejk** [reply]
$5000 is just an example. You can buy a device for just $1000 or even less. :)

Laptops are just this expensive if you want a workhorse that can handle any client's work. (or if you want a good Macbook Pro)

**Comment by MarcusIrmscherSSW** [comment]
Nice job Stephan 😀

**Comment by BenDunkerleySSW** [comment]
A once confusing topic made much clearer, definitely a rule + video many people will be referring to in the future 😄

**Comment by apforsyth** [comment]
great explanation Stephan!

**Comment by AntPolkanov** [comment]
Love the video. Love salary sacrifice. 
Should start using it!

**Comment by zacharykeeping** [comment]
Great video - very easy to understand.

**Comment by leahy268** [comment]
Nice easy to understand video @stephanfakossw.  Look forward to seeing more of this.  Great way to get dry accounting topics through.

**Comment by cvionnet** [comment]
Great video, Stephan👍!
It's really helpful for better understanding the concept of salary sacrifice.
Just wondering, are there any cons?

**Reply by stephanfakossw** [reply]
Not that I can think of mate... However, there are limits on what you can salary sacrifice and how many times.

**Comment by YalalCakirSSW** [comment]
Great video @stephanfakossw !
Thanks to him, I learned this yesterday and when I saw the video, I understood even better. Good examples.

**Comment by bettybondoc** [comment]
Great video! I used to brush it off cause I didn't think there were any benefits to it. Apparently, I could be saving money — sadly I just upgraded my tech recently 🥲

**Comment by ricksu978** [comment]
Great video! It's wonderful to see financial figures being explained through charts and diagrams.

**Comment by synnoveisabel** [comment]
Great video Stephan, it gave a good explanation 😄

**Comment by amankumarrr** [comment]
Great explanation! Now, it'll be convenient to reference this video for those exploring salary sacrifice for the first time.

**Comment by AttackOnMorty** [comment]
Awesome video! Ordering a new MBP...

**Comment by isaaclombardssw** [comment]
Easy to understand explanation of a topic I had no idea about previously – great content.

**Comment by tomek-i** [comment]
I don't understand why the salary sacrifice laptop needs to be branded. 
It would make sense if the company provides it eg. they paid for it 100% and hand it out or "gift" it to you.


By the way, what if my house actually has a batter? :-D

**Reply by steven0x51** [reply]
It has to be portable and has a battery, such as an electric car😂

**Comment by tombui99** [comment]
It's an awesome benefits! I've used it 3 times already!

**Comment by tkapa** [comment]
Great video! Salary sacrificing is a great tool.
(I'm gonna go salary sacrifice something now)

**Comment by chrisschultzssw** [comment]
Well done Stephan! You got me thinking about what I should buy next...

**Comment by christoment** [comment]
Thanks for the great video @stephanfakossw !
One thing I don't like is to put stickers on my devices, I like my device clean 😭

**Comment by JudyMao99** [comment]
Love the video @stephanfakossw 🥳
It gave me a clear overview of Salary Sacrificing, an idea I have never heard about. It is a good method to pay less tax.

**Comment by sethdaily** [comment]
Good video from Stephan and good employee perk.

**Comment by bahjatalaadelssw** [comment]
Useful benefit and it was great seeing Stephan in front of the camera 🤩 Great video!

**Comment by Ben0189** [comment]
Great video from Stephan, thanks for the explanation☺️

**Comment by KostaMadorsky** [comment]
Awesome video, Stephan!
I would remove the LED statue though, it's quite distracting IMHO.

Would be great to get some metal engraved stickers for phones and smaller rectangular stickers for laptops.

**Comment by vladislav-kir** [comment]
Awesome video from Stephan!

The breakdown was super clear and helped understand how much we can save 🤩

**Comment by Harry-Ross** [comment]
Great video with some really important information - a super simple explanation of salary sacrificing that I didn't fully understand before!

**Comment by wicksipedia** [comment]
Updated: this was something that happened 5+ years ago and is no longer legal 
~The only thing that isn't covered in the video it that since your taxable income is lowered -> your super contributions will be lowered as well.~

~Not a huge deal with a laptop but more so with a car since it is a larger value item.~

This can have other benefits like dropping you to a lower tax bracket or be exempt from certain things.

When I worked for a Novated Leasing company, we got a bug report that the system wasn't calculating tax properly. When a driver increased the amount of driving he did in a year (more expenses) -> his pay went up! WTF!?

Turns out, since his deductions increased - he was no longer required to pay the Medicare levy surcharge which was like 1.5% of his salary. #ItsAFeatureNotABug

**Reply by stephanfakossw** [reply]
Hey mate, that's not correct on lower super contributions... 
Lowering Super because of lower adjusted gross salary was a company choice - typically made by bad employers (it was never the default). 
Few years back, ATO made it illegal for employers to dip into employee's SGC (i.e. employers need to pay Super on the gross amount before any pre-tax deductions - see the table in the rule for the math).

**Reply by steven0x51** [reply]
Thanks for sharing, good to know that lol

**Reply by wicksipedia** [reply]
At the time, our systems supported it if companies wanted to do it.

Didn't know that they changed the regs

**Comment by steven0x51** [comment]
Very clear explaination, awesome video, Stephan! 
Just checked the allowance for salary sacrificing a phone is up to $2k, but nowadays an iPhone with decent storage is more than $2k.

**Comment by JackDevAU** [comment]
Awesome video @stephanfakossw 🚀

**Comment by Marxoz** [comment]
Good video Stephan! Thanks for the explanation.

But what if my house has a battery?? 😂

"Do you use consistent salary terminology?" | https://www.ssw.com.au/rules/salary-terminology/ - Might be worth mentioning that any super adjustments comes from your salary if this terminology is used. Other terminologies are also accepted and understood, but have a different meaning.

**Reply by GordonBeeming** [reply]
@Marxoz you don't live in 🇿🇦 South Africa 😉

**Comment by Levijj22** [comment]
Great video Stephan!

Love Salary Sacrificing! It's not very common to be able to pay it off over a 12 month period, thanks Cogans! 😁

**Comment by CoffeeBugHub** [comment]
Very valuable video, really get to understand how the calculation works this time! Considering Salary sacrifice now.

**Comment by jernejk** [comment]
I love the video and love the rule, awesome job @stephanfakossw. If not the best CTF, it is one of the best ones. :) (short and sweet as well)

I also love the idea of promoting SophieBot more as stickers and other branding material. :)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/e4da9082-3005-4e4d-9a25-e245806ae7d1)


I don't know if this is the right channel for this. While I love the idea of branding, skins can be a bad idea if not executed really well. Once an A4-ish sticker starts to degrade, it's obvious and hard to replace.

This is my phone with a case for a good grip. This kind of case will quickly rip away most of the stickers (I tried).
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/05fa70cf-93f1-4f19-8edd-941136a0df22)

I also have a custom skin for my laptop, so I can put on stickers and rip them away when I sell the laptop in a couple of years.
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/99f4508b-d111-4163-b3d2-bde5cdf53acf)

While it does protect the back well, looks awesome and makes reselling easier (no scratches due to the removal of stickers), the stickers don't stick as well. :(

And my tablet has 2 different cases.
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/571dc067-efae-4500-933b-e92345e72b13)

Stickers are on the case, so you wouldn't ever see the stickers on the device. However, the stickers do stick better, and I love the SophieBot sticker. :)

**Reply by leahy268** [reply]
The big plastic stickers that we use are great.  Even a few years on they still look great. I used them for all the school laptops I used to look after.  They stay on, protect the device from scratches and when you want them off they peel off no prob. Then only thing you might need is an alcohol swab at the end to clean up the edges.

**Comment by SamBWagner** [comment]
salary sacrifice is cool. Branding my device is not cool

**Reply by drwharris** [reply]
But is it your device 🤔

Company pays for the device outright so that you can use it while you pay the company off.

**Comment by UlyssesMaclaren** [comment]
I've been salary sacrificing my devices for ages since I love the massive savings on what would otherwise be a prohibitively expensive laptop/phone!

Love this system 🥰

**Comment by jackreimers** [comment]
Awesome video and explanation, salary sacrificing for devices is a huge perk!
Personally not a fan of branding the laptop or sticking any kind of sticker on an expensive device that I'm paying for in the end, even if it's cheaper.

A solution for those who don't want to do that, which I'd say is the majority, could be getting some branded laptop cases or pouches.

**Reply by leahy268** [reply]
Of course it's a personal choice.  The big plastic stickers we use come off well at the end. Just need an alcohol swab to clean up the last of the stickyness.  But they look so good when there are lot's of us in the same place..

**Reply by drwharris** [reply]
Dont forget, you dont own it until you pay it off. I dont see the issue with a small sticker (Which as Warwick points out, comes off easily) to say thank you for helping you pay for your new bit of kit.

**Comment by ozair-ashfaque** [comment]
Great explanation, thanks Stephan

**Comment by PennyWalker** [comment]
I love salary sacrifice, it's allowed me to get shiny new things every year for years - and now I can see how much that's prob also saved me in tax! This was a great video, it was so easy to understand after watching, thanks Steph! <3

**Comment by drwharris** [comment]
Great video Stephan!

**Comment by jerryluossw** [comment]
Unfortunately, it doesn't work in China 😭😭😭

**Comment by Aibono1225** [comment]
Very clear explanation! It would be super cool if we have it in China 😭

**Comment by jimmidier** [comment]
Great video, nice done Stephan. But I wonder how it works in China

**Comment by sylhuang** [comment]
Love the video and clear explanation! Hope we could get something similar in Hangzhou Office 😋

**Comment by tino-liu** [comment]
Bummer China doesn't get this perk :( But cool video!

**Comment by danielmackay** [comment]
Didn't know we could salary sacrifice a smart-watch.  Wish I had have known that 1 month ago... 😢

**Comment by AnkyChe** [comment]
Nice video, both the explanation and calculation are very clear. Hope we can figure out a way for Chinese employees to get this perk.

**Comment by KristenHu** [comment]
Awesome rule, but when will we have it in Hangzhou office🥺

**Comment by joshbermanssw** [comment]
Heard about it before but never knew entirely what it was. Great Explanation!

**Comment by kenzengssw** [comment]
Great video! The rule is really beneficial. Unfortunately, as an intern, I won’t be able to participate in this perk😩.

**Comment by BrookJeynes** [comment]
Great video @stephanfakossw, very clear explanations

**Comment by TobyChurches** [comment]
Great explanation and example! Ill be sure to take full advantage of this now 💡

**Comment by babakamyljanovssw** [comment]
Great video and explanation, looking forward that it would be applicable to Hangzhou office too.

**Comment by matt-goldman** [comment]
Fantastic video, thanks Stephan!

**Comment by JackDuan123** [comment]
Sorry for missing this one, Salary sacrifice is pretty awesome, but I think there are 2 points that needs to be considered or addressed:

1. If we don't buy anything, we still get more cash back, so that's why doing salary sacrifice annually might not be the best financial option, or there are better ways to spend your money.
2. You can, sell that for some money, but it's also some work to do, is it really worth the time? Or, in other word, there should be better way to spend time

I will only use Salary sacrifice when I really need new devices.

**Comment by wingy3181** [comment]
how does single touch payroll impact this?
My gross salary inclusive of the deduction is now reported to the ATO
so feels like i lose the benefit of salary sacrificing

**Reply by stephanfakossw** [reply]
Great question! 
The ATO takes into account your salary sacrifice (pre-tax) deductions when calculating your taxable income. Savings on tax still apply!
